releasenotes/notes/ha-agent-integrations-f8923fd651727669.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
```

Comment:
@jedupau Let's not have a release note for this, since we don't want to make this feature too visible yet.